CONTRIBUTING.md

```diff
@@ -17,6 +17,13 @@ a patch, please do the following:
 * Limit the first line of your commit message to 50 characters
 * Describe the bug you found or the feature that is missing from the project
 * Describe how your patch fixes the bug or improves the project
+* Make sure the code keeps working after each commit, and does not require any
+  further commits to fix it.
+* Make sure each commit contains only one complete logical change, no more, no
+  less.
+* When adding a new module, a class, or a function always document them.
```

Comment:
Small nit: "Always document each new module, class, or function."

Reply:
Yes, that's better. Thanks, Major!